application/F3DOptionsParser.h

```diff
@@ -88,10 +88,10 @@ class F3DOptionsParser
   F3DOptionsParser();
   ~F3DOptionsParser();
 
-private:
   F3DOptionsParser(F3DOptionsParser const&) = delete;
   void operator=(F3DOptionsParser const&) = delete;
 
+private:
```

Comment:
not sure why though

Reply:
https://stackoverflow.com/questions/18931133/must-a-deleted-constructor-be-private